.circleci/config.yml

```diff
@@ -0,0 +1,87 @@
+# These environment variables must be set in CircleCI UI
+#
+# DOCKERHUB_REPO - docker hub repo, format: <username>/<repo>
+# DOCKER_EMAIL   - login info for docker hub
+# DOCKER_USER
+# DOCKER_PASS
+#
+version: 2
+jobs:
+  build:
+    docker:
+      # base image
+      - image: ubuntu:16.04
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run:
+          name: Install essential packages
+          command: |
+            apt-get update
+            apt-get install -y ca-certificates curl build-essential make git
+
+      - run:
+          name: Create version.json
+          command: |
+            printf '{"commit":"%s","version":"%s","source":"https://github.com/%s/%s","build":"%s"}\n' \
+            "$CIRCLE_SHA1" \
+            "$CIRCLE_TAG" \
+            "$CIRCLE_PROJECT_USERNAME" \
+            "$CIRCLE_PROJECT_REPONAME" \
+            "$CIRCLE_BUILD_URL" > version.json
+      - store_artifacts:
+          path: version.json
+      # FIXME: should use an image w/ docker installed by default
+      - run:
+          name: Install Docker
+          command: |
+            set -x
+            curl -L -o /tmp/docker-$DOCKER_VERSION.tgz https://download.docker.com/linux/static/stable/x86_64/docker-$DOCKER_VERSION.tgz
+            tar -xz -C /tmp -f /tmp/docker-$DOCKER_VERSION.tgz
+            mv /tmp/docker/* /usr/bin
+      - run:
+          name: Build deployment container image
+          command: docker build -t app:build .
+      - run:
+          name: Create Docker image shasum256
+          command: |
+            docker images --no-trunc |
+            awk '/^app/ {print $3}' |
+            tee docker-image-shasum256.txt
```

Comment:
@jbuck do we need to do this or based on https://circleci.com/gh/mozilla/fxa-oauth-server/413 the digest log is enough information for deployment?

Reply:
This can be removed, the `docker push` log is what we use to match